### DISCUSSION TITLE
Is this signal FSK or PSK??

### DISCUSSION THREAD

**Discussion by emre2blue** [jopohl/urh#1090] [General]
Hello, I'm using URH in windows 11 with RTL-SDR. I'm working on my car with TPMS, I've successfully capture the signal and decoded in UHR. And I'm a maker. I want to capture this signal in  CC1101 also. Sensor catalog says that it is FSK modulated 433.92Mhz - 76khz deviation , 9600 baud rate. And Manchester coded. All seems fine . But is this signal really FSK ???? 

![URH_FSK_PSK](https://github.com/jopohl/urh/assets/14080943/7e16a5fb-243c-40e4-9f31-9e89a902732c)
![URH_Decode](https://github.com/jopohl/urh/assets/14080943/0ff2e5b5-2b38-4eab-b7f4-1153d5a61b32)

UHR Decoded the signal without problem. And data is there but how is this a FSK signal. I'm asking because when I want to capture it with CC1101, I get random bits and I could not manage the capture any real data since I started the project. But every time I use UHR it captures perfectly

![SensorData](https://github.com/jopohl/urh/assets/14080943/00ab0b2e-fb9b-4ad7-8b40-84f56968ece7)

Here is the data for UHR [https://drive.google.com/drive/folders/1yoGfmySBvS7HPjrfBjRb0RMwVOsWrdtn?usp=sharing](url)

**Comment by emre2blue** [comment]
Additional information : 

![WhatsApp Image 2023-11-22 at 08 58 32_6e2a5186](https://github.com/jopohl/urh/assets/14080943/8be1bbd7-6829-4212-8357-3e2cc4cba195)

if you look at the wave from of BPSK. it is almost same as the data I've capture from the URH. how can it solve the wave form as FSK???

also I've done a spectrum analyzer in URH.  When I check the form it is like FSK this time. 

![WhatsApp Image 2023-11-22 at 09 00 22_b1d0821c](https://github.com/jopohl/urh/assets/14080943/fbd4ab6e-d37b-421e-96e9-2f3d8adba35c)
![WhatsApp Image 2023-11-22 at 09 08 18_77d440e6](https://github.com/jopohl/urh/assets/14080943/c53ab842-197d-425e-af35-ba5cdcdea959)

I'm so confused. Please help :)

**Reply by kd2wtr** [reply]
I'm not sure. It may be that URH is wrong, but that depends on whether the signal's information is decodable. Try something like FLDIGI. With the modes you think it might be. It might be a weird kind of FSK, I wouldn't worry about it-if it ain't broke don't fix it :)
